pkg/controller/resourcequota/resource_quota_controller_test.go

```diff
@@ -23,11 +23,32 @@ import (
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/resource"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
 	"k8s.io/kubernetes/pkg/client/unversioned/testclient"
 	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/util/sets"
 )
 
+func newResourceQuotaController(kubeClient client.Interface) *ResourceQuotaController {
+	resourceQuotaUsageRegistry := NewDefaultUsageFuncRegistry(kubeClient)
+	resourceQuotaControllerFactory := NewMonitoringControllerFactory(kubeClient)
+	groupKindsToMonitor := []unversioned.GroupKind{
+		unversioned.GroupKind{Group: "", Kind: "Pod"},
+		unversioned.GroupKind{Group: "", Kind: "Service"},
+		unversioned.GroupKind{Group: "", Kind: "ReplicationController"},
+		unversioned.GroupKind{Group: "", Kind: "PersistentVolumeClaim"},
```

Comment:
Maybe I'm missing something, but it seems to me `Secrets` and `ResourceQuota` are missing in the array.